lib/radis-lambda/download_hitran.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Download some molecules from HITRAN data, parse & cache them, to make 
+sure the example notebooks run fast.
+"""
+
+from radis.io.hitran import fetch_hitran
+
+for molecule in [
```

Comment:
@suzil do we have an equivlaent of this pre-download script on the new machine?

Reply:
I ran the script on the server now so it should be working as expected from now.

Reply:
Code for the server added in this PR: https://github.com/suzil/radis-app/pull/568

Reply:
Nice ! I'm adding a `download_geisa.py` script there; but I do not know where to call it (maybe you have to run it on the server bash ? )

Reply:
Yes, I'll run it manually.